### ISSUE TITLE
Create View (Material)

### ISSUE THREAD

**Issue by BeckTJ** [BeckTJ/RLRMDB#4]

**Comment by BeckTJ** [comment]
The ability to view material and add material to the database is started. No checks have been implemented. A new issue for adding them.

**Comment by BeckTJ** [comment]
Code will need to be refactored once a UI is introduced. Once update is complete Create the UI. Need understand the process before continuing back-end development.

**Comment by BeckTJ** [comment]
Material View Is operational. New issue will be created with more in-depth requirements.
